src/components/WindowSideBarInfoPanel.js

```diff
@@ -1,65 +1,83 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import Divider from '@material-ui/core/Divider';
 import Typography from '@material-ui/core/Typography';
 import { withStyles } from '@material-ui/core/styles';
+import LabelValueMetadata from './LabelValueMetadata';
 import ns from '../config/css-ns';
 
 /**
  * WindowSideBarInfoPanel
  */
 class WindowSideBarInfoPanel extends Component {
-  /**
-   * manifestLabel - get the label from the manifesto manifestation
-   * @return String
-   */
-  manifestLabel() {
-    const { manifest } = this.props;
-
-    if (manifest.manifestation) {
-      return manifest.manifestation.getLabel().map(label => label.value)[0];
-    }
-    return '';
-  }
-
-  /**
-   * manifestDescription - get the description from the manifesto manifestation
-   * @return String
-   */
-  manifestDescription() {
-    const { manifest } = this.props;
-
-    if (manifest.manifestation) {
-      return manifest.manifestation.getDescription().map(label => label.value);
-    }
-    return '';
-  }
-
   /**
    * render
    * @return
    */
   render() {
-    const { classes, t } = this.props;
+    const {
+      canvasDescription,
+      canvasLabel,
+      canvasMetadata,
+      classes,
+      manifestDescription,
+      manifestLabel,
+      manifestMetadata,
+      t,
+    } = this.props;
     return (
       <div className={ns('window-sidebar-info-panel')}>
         <Typography variant="h2" className={classes.windowSideBarH2}>{t('aboutThisItem')}</Typography>
-        <Typography variant="h3" className={classes.windowSideBarH3}>{this.manifestLabel()}</Typography>
-        <Typography variant="body2">{this.manifestDescription()}</Typography>
+        {canvasLabel
+          ? <Typography variant="h3" className={classes.windowSideBarH3}>{canvasLabel}</Typography>
+          : null
```

Comment:
Happy to entertain better ways of doing this.  It feels sub-optimal, but I think we should avoid empty headings hanging out if possible.

Reply:
AFAIK you can return all kinds of falsy values if you don't want to render anything. So undefined would be valid too. This means you can also do:

```javascript
{ canvasLabel && 
  <Typography variant="h3" className={classes.windowSideBarH3}>{canvasLabel}</Typography>
}
```

Saves one line! :sunglasses: